src/app/child-window-handler.ts

```diff
@@ -153,6 +153,7 @@ export const handleChildWindow = (webContents: WebContents): void => {
             newWinOptions.alwaysOnTop = mainWindow.isAlwaysOnTop();
             newWinOptions.frame = true;
             newWinOptions.winKey = newWinKey;
+            newWinOptions.fullscreenable = false;
```

Comment:
@johankwarnmarksymphony This should be just `fullscreen`. setting `fullscreenable: false` pop-out cannot be made fullscreen
```suggestion
            newWinOptions.fullscreen = false;
```

Reply:
Great catch I will update